security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -15,6 +15,13 @@ For an overview of the File Integrity Operator, see xref:../../security/file_int
 
 To access the latest release, see xref:../../security/file_integrity_operator/file-integrity-operator-updating.adoc#olm-preparing-upgrade_file-integrity-operator-updating[Updating the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-1-3-5"]
+== OpenShift File Integrity Operator 1.3.5
+
+The following advisory is available for the OpenShift File Integrity Operator 1.3.5:
+
```

Comment:
Do we need to add any statements like we did on line 44 for 1.3.3?

Reply:
I wasn't sure if we were upgrading based on underlying dependencies, I wanted to check first. 

I'll update all 3 release notes PRs to include that statement 👍